examples/react-apollo/cypress/e2e/1-sign-up/existing-email.cy.ts

```diff
@@ -0,0 +1,11 @@
+import { faker } from '@faker-js/faker'
+
+context('Failed attempt to sign up with an email already present in the database', () => {
+  it('shoud raise an error when trying to sign up with an existing email', () => {
+    const email = faker.internet.email()
+    const password = faker.internet.password(10)
+    cy.signUpEmailPassword(email, password)
```

Comment:
I'd put some assertions here to confirm that registration was successful at first attempt.